src/test/java/nl/tudelft/pixelperfect/gamestates/PlayStateTest.java

```diff
@@ -45,11 +47,13 @@ public void setUp() {
     mockScheduler = mock(EventScheduler.class);
     mockObserver = mock(Spatial.class);
     mockSpaceship = mock(Spaceship.class);
-    mockHeadsUp = mock(DebugHeadsUpDisplay.class);
+    mockDebugUp = mock(DebugHeadsUpDisplay.class);
+    mockHeadsUp = mock(GameHeadsUpDisplay.class);
     mockGame.setGameObserver(mockObserver);
     mockGame.setScheduler(mockScheduler);
     mockGame.setSpaceship(mockSpaceship);
     mockGame.setHeadsUpDisplay(mockHeadsUp);
+    mockGame.setDebugDisplay(mockDebugUp);
     testSettings = Settings.getInstance();
```

Comment:
Verified this again, it has to be a mistake, it's used in two tests directly after the warning.